tfjs/scripts/build-npm.sh

```diff
@@ -27,5 +27,8 @@ mkdir dist/miniprogram
 cp dist/tf.min.js dist/miniprogram/index.js
 cp dist/tf.min.js.map dist/miniprogram/index.js.map
 
+# Copy the root README over.
+cp ../README.md .
```

Comment:
-f to force in case there is already a readme, or do you prefer it to fail?

Reply:
It won't fail if there's already a readme, cp just overwites.